pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -77,11 +78,13 @@ type Deployer struct {
 	accessor      access.Accessor
 	logger        log.Logger
 	debugger      debug.Debugger
+	imageLoader   loader.ImageLoader
 	statusMonitor status.Monitor
 	syncer        sync.Syncer
 
 	podSelector    *kubernetes.ImageList
 	originalImages []graph.Artifact
```

Comment:
can you add a comment about what `localImages` and `originalImages` hold?

Reply:
done